docs/receptor_mesh.md

```diff
@@ -0,0 +1,66 @@
+## Receptor Mesh
+
+AWX uses a [Receptor](https://github.com/ansible/receptor) mesh to transmit "user-space" unified jobs:
+ - jobs
+ - ad hoc commands
+ - inventory updates
+
+to the node where they run.
+
+> NOTE: user-space jobs are what carry out the user's Ansible automation. These job types run inside of the designated execution environment so that the needed content is available.
+
+> NOTE: The word "node" corresponds to entries in the `Instance` database model, or the `/api/v2/instances/` endpoint, and is a machine participating in the cluster / mesh.
+
+### Split of Control Plane versus Execution Plane
+
+Instances in the control plane run persistent AWX services (like the web server, task dispatcher, etc.), project updates, and management jobs.
+
+Hybrid instances in the control plane may also run user-space jobs.
```

Comment:
I think maybe move this below the next paragraph so the order is
- Control
- Execution
- Both (aka hybrid)

Reply:
I took your advice to heart, but it kind of ballooned into a larger rewrite. New commit incoming.